lib/reverse_markdown/converters/pre.rb

```diff
@@ -17,6 +17,9 @@ def treat(node, state)
         case node.name
         when 'code'
           node.text
+        when 'text'
+          # Preserve '\n' in the middle of text/words, get rid of indentation spaces
+          ReverseMarkdown.cleaner.remove_leading_newlines(node.text.gsub("\n", '<br>').strip.gsub('<br>', "\n"))
```

Comment:
AFAICS this can be simplified to 

```suggestion
          node.text
```

At least it will pass all provided test cases, right?

Reply:
@shivabhusal, can you look into this PR when you get a chance?

Seems to me like @xijo's proposal works just fine:

```ruby
when 'code', 'text'
  node.text
```